show/main.py

```diff
@@ -2081,6 +2081,98 @@ def ztp(status, verbose):
        cmd += ["--verbose"]
     run_command(cmd, display_cmd=verbose)
 
+#
+# 'bmp' group ("show bmp ...")
+#
+@cli.group(cls=clicommon.AliasedGroup)
+def bmp():
+    """Show details of the bmp dataset"""
+    pass
+
+# 'bgp-neighbor-table' subcommand ("show bmp bgp-neighbor-table")
+@bmp.command('bgp-neighbor-table')
+@clicommon.pass_db
+def bmp_neighbor_table(db):
+    """Show bmp bgp-neighbor-table information"""
+    bmp_headers = ["Neighbor_Address", "Peer_Address", "Peer_ASN", "Peer_RD", "Peer_Port", "Local_Address", "Local_ASN", "Local_Port",
+                   "Advertised_Capabilities", "Received_Capabilities"]
+
+    bmp_keys = db.db.keys(db.db.STATE_DB, "BGP_NEIGHBOR_TABLE|*")
```

Comment:
>keys [](http://example.com/codeflow?start=21&length=4)

Table is above the key/field concept level and is preferred. you do not need to worry about separators if you can use `Table` class.

Reply:
Will discuss offline.